test/index.js

```diff
@@ -44,7 +44,7 @@ describe('eslint-config-hapi', function () {
 
     expect(msg.ruleId).to.equal('strict');
     expect(msg.severity).to.equal(2);
-    expect(msg.message).to.equal('Use the global form of "use strict".');
+    expect(msg.message).to.equal('Use the global form of \'use strict\'.');
```

Comment:
You could use back ticks here to avoid the annoying escape characters. Then it would simply read `Use the global form of 'use strict'`